wp-content/themes/pub/wporg-learn-2024/src/style/style.scss

```diff
@@ -51,3 +51,9 @@ body {
 		padding-top: unset !important;
 	}
 }
+
+.disabled {
+	color: var(--wp--preset--color--charcoal-5) !important;
+	border-color: var(--wp--preset--color--charcoal-5) !important;
+	pointer-events: none;
+}
```

Comment:
I think you can remove this style, since the disabled button has been removed.

Reply:
21f09df